README.md

```diff
@@ -187,6 +187,7 @@ We are following [**Contributor Covenant Code of Conduct**](.github/CODE_OF_COND
   * [C++](strings/reverse_a_string/reverseString.cpp)
   * [Java](strings/reverse_a_string/reverseString.java)
   * [Python](strings/reverse_a_string/reverseString.py)
+  * [Ruby] (strings/reverse_a_string/reverseString.rb)
```

Comment:
This is incorrect please don't mention spaces in between [] and ()
it should be `[Ruby](strings/reverse_a_string/reverseString.rb)`
Please correct previous PR README changes as well